Source/WebCore/dom/DocumentFullscreen.cpp

```diff
@@ -497,17 +499,18 @@ void DocumentFullscreen::exitFullscreen(CompletionHandler<void(ExceptionOr<void>
 void DocumentFullscreen::finishExitFullscreen(Frame& currentFrame, ExitMode mode)
 {
     RefPtr currentLocalFrame = dynamicDowncast<LocalFrame>(currentFrame);
-    if (currentLocalFrame && currentLocalFrame->document() && !currentLocalFrame->document()->fullscreen().fullscreenElement())
+    if (currentLocalFrame && currentLocalFrame->protectedDocument() && !currentLocalFrame->protectedDocument()->fullscreen().fullscreenElement())
```

Comment:
We don't need to call `protectedDocument()` just to do a null check, please revert this part of the change.